cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -30,7 +30,7 @@ import (
 	configv1beta3 "k8s.io/kubernetes/pkg/scheduler/apis/config/v1beta3"
 )
 
-func loadConfigFromFile(logger klog.Logger, file string) (*config.KubeSchedulerConfiguration, error) {
+func LoadConfigFromFile(logger klog.Logger, file string) (*config.KubeSchedulerConfiguration, error) {
```

Comment:
I think we should annotate the export function if we need it.

Reply:
Ah, you are right. I didn't add a comment on top of the function. Thanks for noticing. I will add it :pray:

Reply:
@aimuz I have added the comment.

Reply:
/lgtm